src/geometry/CTiglPoint.cpp

```diff
@@ -177,4 +178,37 @@ void CTiglPoint::getMinMax(double & min, double & max) const
     }
 }
 
+bool CTiglPoint::operator==(const CTiglPoint& aPoint) const
+{
+    if (x == aPoint.x && y == aPoint.y && z == aPoint.z) {
+        return true;
+    }
+    return false;
+}
+
+bool CTiglPoint::isNear(const CTiglPoint& aPoint, double epsilon) const
+{
+    double d2 = distance2(aPoint);
+    double d = sqrt(d2);
+    if( d < epsilon) {
+        return true;
+    }
+    return false;
+
+}
+
+void CTiglPoint::normalize()
+{
+    if (norm2() == 0) {
+        LOG(WARNING)
+            << "CTiglPoint::normalize: The norm is 0, so it's impossible to normalize, 0 length vector is returned."
+            << std::endl;
```

Comment:
Remove std::endl from the log line. It is automatically appended by the logger.

Reply:
ok
done
commit: c2d9b9084f9cf34caa645b4b95a536bd62a6541b